src/components/NewMessage/NewMessage.vue

```diff
@@ -644,14 +645,24 @@ export default {
 		},
 
 		async handleEdit() {
+
+			if (this.messageToEdit.messageParameters.length !== 0) {
```

Comment:
this also matches mentions :-P

Reply:
still didn't make my last scan on the code :')